interpreter/interpretable.go

```diff
@@ -1277,6 +1279,38 @@ func (a *evalAttr) Resolve(ctx Activation) (any, error) {
 	return a.attr.Resolve(ctx)
 }
 
+type evalWatchConstructor struct {
```

Comment:
Where is this allocated and used?

Reply:
Terribly sorry, that somehow came from https://github.com/google/cel-go/pull/645. I've double-checked to make sure there's no other code from other branches.